vertical-pod-autoscaler/docs/faq.md

```diff
@@ -244,3 +245,15 @@ The `--webhook-labels` parameter for the VPA admission-controller can be used to
 When running in EKS with Cilium, the EKS API server cannot route traffic to the overlay network. The VPA admission-controller
 Pods either need to use host networking or be exposed through a service or ingress.
 See the [Cilium Helm installation page](https://docs.cilium.io/en/stable/installation/k8s-install-helm/) for more info.
+
+### Why does VPA fail to create a Pod when resource limits are explicitly set to 0?
+VPA currently handles requests=0 and limits=0 inconsistently with Kubernetes, leading to invalid Pod configurations. Here’s the breakdown:
+
+| Scenario                | Kubernetes Behavior                          | VPA Behavior                                  |
+|-------------------------|----------------------------------------------|----------------------------------------------|
+| `requests=0`+`limits=0`| Disables resource fields (BestEffort QoS)    | VPA recommends non-zero requests, which is invalid, and this will change the QoS to `Burstable`|
```

Comment:
I think it's worth mentioning that limits is also set to 0
non-zero requests is fine, as long as it's above limits, but in our scenario we get non-zero requests **and** limits set to zero.